docs/meshcloud.payment-methods.md

```diff
@@ -0,0 +1,94 @@
+
+2) A budget payment method (indicated via `COST_LIMITATION`), meaning costs made by the meshTenants in your meshProjects will be charged based on the monetary budget amount that is set on this payment method.
+
+![Payment Methods Lifecycle](assets/payment_methods/payment_method_overall_lifecycle.png)
+
+## The Payment Method Lifecycle: Creation
+
+The first step in reaping the benefits of payment methods is creating them. A few important things must be considered about payment methods before creating them
+
+1. A payment method is **always** scoped to a single meshCustomer. This means it is not possible to create a global payment method that can be used for all meshCustomers. It is also not possible to re-use a single payment method across multiple meshCustomers.
+2. A payment method's identifier is globally unique. This means you cannot create a payment method with the same identifier for multiple meshCustomers. Make sure to use an identifier that will not collide in other meshCustomers.
+
+### Creating a payment method via the meshPanel
+
+One way of creating payment methods is via the meshPanel. To do so, make sure that you have an account with the 'Partner Admin' role. Navigate to the Admin area and follow these steps:
+
+1. Click on 'Customers' on the left.
+2. Find a customer you want to create a payment method for and click on the 'Customer Payment Methods' icon on the right.
+
+    ![List Payment Methods](assets/payment_methods/customer_list_payment_methods.png)
+
+3. Click on 'Create Payment Method' at the top right.
+4. Enter a name and identifier for the new payment method.
+5. Choose a type for the payment method. Read more here on what type to choose and what the individual differences are.
+6. (This is optional) Set a maximum amount of EUR on the payment method to indicate the remaining budget for this year.
+7. (This is optional) Set an expiration date for the payment method. This is especially useful when a budget expires, e.g. at the end of the accounting year.
+8. Additionally, you can enter custom settings to inject metadata into the payment method. This is useful when you want to enhance the payment method with organizational details like the cost center number or the business unit.
+9. Click 'Save' and your new payment method will be available to the meshCustomer it was created in!
+
+### Creating a payment method via the meshstack API
+
+As automation matters, there is also the possibility to create payment methods via the meshstack API. To find the API documentation, open your meshstack and click on 'API' at the bottom of the meshPanel. A new page will open and on the left-hand side, you should see `meshPaymentMethod` listed under `meshObject Import`. Click on it and read the documentation to understand how you can create a payment method via the API.
+
+## The Payment Method Lifecycle: Assigning to Projects
```

Comment:
d: should we use `meshProject` instead of `project` as a term everywhere? At least for `meshCustomers` we do (mostly).

Reply:
Will change to meshProject

Reply:
Done everywhere where it made sense (sometimes we quote literal stuff from the panel so it doesn't make sense there)